mypyc/common.py

```diff
@@ -15,6 +15,8 @@
 
 TOP_LEVEL_NAME = '__top_level__'  # Special function representing module top level
 
+MAX_CHILDREN = 2  # Number of children for a class to trigger fast path in isinstance() checks.
```

Comment:
What about making the name more descriptive, such as `FAST_ISINSTANCE_MAX_SUBCLASSES`?

Reply:
👍